distributed_index_tests/lib/syncgateway.py

```diff
@@ -10,111 +10,16 @@
 
 class SyncGateway:
 
-    def __init__(self, host_info, db_name):
-        self.public_port = 4984
-        self.admin_port = 4985
-        self.ip = "http://{}".format(host_info["ip"])
-        self.host_name = host_info["name"]
-        self.db = Database(self.ip, self.admin_port, self.public_port, db_name)
+    def __init__(self, target):
+        self.ip = target["ip"]
+        self.url = "http://{}:4984".format(target["ip"])
+        self.host_name = target["name"]
 
     def info(self):
-        return requests.get("{0}:{1}".format(self.ip, self.public_port))
+        return requests.get(self.url)
```

Comment:
it would be nice to have automatic validation of 200OK or positive response. also please use exception if 4xx or 5xx response.